anaUltraScurve.py

```diff
@@ -304,8 +320,7 @@ def plotAllSCurvesOnCanvas(vfatHistos, vfatHistosPanPin2=None, obsName="scurves"
     if options.performFit:
         fitter = ScanDataFitter(
                 calDAC2Q_m=calDAC2Q_Slope, 
-                calDAC2Q_b=calDAC2Q_Intercept,
-                isVFAT3=options.isVFAT3
+                calDAC2Q_b=calDAC2Q_Intercept
```

Comment:
ah we should keep link 308 but change it to:

```python
isVFAT3=isVFAT3
```

Otherwise the `ScanDataFitter` class instance will not behave correctly.